chunker/pom.xml

```diff
@@ -2,7 +2,7 @@
     <parent>
         <artifactId>illinois-cogcomp-nlp</artifactId>
         <groupId>edu.illinois.cs.cogcomp</groupId>
-        <version>3.0.61</version>
+        <version>3.0.63-SNAPSHOT</version>
```

Comment:
Yes -- am deploying snapshots so I can work on the project(s) that need(s) these enhancements without delay. I'd prefer not to have to change the version back to the previous one just for PRs.